tutorials/include/detray/tutorial/square_surface_generator.hpp

```diff
@@ -63,7 +63,7 @@ class square_surface_generator final
                     typename detector_t::surface_lookup_container &surfaces,
                     typename detector_t::transform_container &transforms,
                     typename detector_t::mask_container &masks,
-                    typename detector_t::geometry_context ctx = {}) const
+                    typename detector_t::geometry_context ctx = {})
```

Comment:
No, you are right, I fully mean to revert this. This happened in an earlier version of the ```cylinder_portal_generator``` where it could both be filled with explicit surface data from the outside or do the automatic fit alternatively. I later decided that was nonsense, as the generic ```surface_factory``` can already be used to  build specific portals

Reply:
Ok, looks like I can't do that. The portal_cylinder_generator is still recording a few results internally